src/github.ts

```diff
@@ -173,7 +177,7 @@ const createGitHubRepository = TE.tryCatchK<Error, [CreateGitHubRepositoryParams
           ...defaults,
           ref: `heads/${branch}`,
           sha: commit.sha,
-          force: true,
+          force: force,
```

Comment:
imo. Below I prefer the style of 😄 

```suggestion
          force,
```